.github/ISSUE_TEMPLATE/SlicingCrash.yaml

```diff
@@ -5,38 +5,24 @@ body:
 - type: markdown
   attributes:
     value: |      
-       ### ✨Try our improved Cura 5.7✨
-       Before filling out the report below, we want you to try the latest Cura 5.7. 
-       This version of Cura has become significantly more reliable and has an updated slicing engine that will automatically send a report to the Cura Team for analysis. 
-       #### [You can find the downloads here](https://github.com/Ultimaker/Cura/releases/latest) #### 
-       If you still encounter a crash you are still welcome to report the issue so we can use your model as a test case, you can find instructions on how to do that below.      
+       ### ✨Are you stuck? Have you tried these two things? ✨
+       1- Are you on a Cura version lower than Cura 5.7? We really recommend updating because it resolves a lot of slicing crashes!
+       2- Have you tried fixing the model with software that repairs 3d files and makes them watertight? 
+       Are you seeing spots and dots on your model? That is Cura indicating that your model is not watertight.  
        
-       ### Project File
-       **⚠️ Before you continue, we need your project file to troubleshoot a slicing crash.**
-       It contains the printer and settings we need for troubleshooting. 
+       If you still encounter a crash you are welcome to report the issue so we can use your model as a test case. 
+       You can find instructions on how to share your model in a Package for Technical Support below.        
        
-       ![Alt Text](https://user-images.githubusercontent.com/40423138/240616958-5a9751f2-bd34-4808-9752-6fde2e27516e.gif)
-       
-       To save a project file go to File -> Save project. 
-       Please make sure to .zip your project file. 
-       For big files, you may need to use [WeTransfer](https://wetransfer.com/) or similar file-sharing sites.
-       
-       🤔 Before you share, please think to yourself. Is this a model that can be shared?
-           Unfortunately we cannot help if this file is missing. 
-           Do you have the project file? Than let's continue ⬇️
-
-       ### Questions
+       🤔 Before you share, please think to yourself. Is this a model that can be shared on the internet?
+       **Unfortunately, we cannot help if this file is missing.** 
+     
```

Comment:
Maybe add a line where we encourage people to search for already reported issues first (if not _too_ old) -- and then post there.

Reply:
We do that for the bugs and features, but those are harder for Slicing Crashes where the symptoms are all the same.